README.md

```diff
@@ -40,6 +40,42 @@ Task {
 }
 ```
 
+#### Offline initialization
+
+If you'd like to initialize Eppo's client without performing a network request, you can pass in a pre-fetched configuration JSON string.
```

Comment:
Note to self: mention that if using a packaged and stale configuration it might not be desirable to add an assignment logger to the client; for this case better to do so when initializing from remote configuration.

For others who are passing a configuration over the wire they are likely satisfied with an application logger at this step.